{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":20,"msg":"Sync server version: 1.3.8 2022-02-25 10:58:11 -0600 50659ecda06a8b79a41f24c1fdff05187324402b","time":"2024-11-05T19:47:36.782Z","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":30,"msg":"Acquiring actors for session 1730836057: ACTOR1","time":"2024-11-05T19:47:36.792Z","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":30,"msg":"Session 1730836057 with label \"feature-Oneapp\" started at 2024-11-05 19:47:36 UTC","time":"2024-11-05T19:47:36.807Z","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":30,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.4704993Z","msg":"Actor 27587 of type ACTOR1 was acquired by test session 1730836057","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":20,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.4955979Z","msg":"DEBUG: Test actor JAR versions:","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":20,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.5005926Z","msg":"DEBUG:   opentest-appium-0.0.1.jar: 2022-02-25 10:58 -0600 0.0.1-50659ecda06a8b79a41f24c1fdff05187324402b-dirty","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":20,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.5015918Z","msg":"DEBUG:   opentest-base-0.0.1.jar: 2022-02-25 10:58 -0600 0.0.1-50659ecda06a8b79a41f24c1fdff05187324402b-dirty","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":20,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.504593Z","msg":"DEBUG:   opentest-selenium-0.0.1.jar: 2022-02-25 10:58 -0600 0.0.1-50659ecda06a8b79a41f24c1fdff05187324402b-dirty","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":40,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.5125962Z","msg":"WARN: Failed to log extension JARs","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":10,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.5136181Z","msg":"TRACE: Firing SESSION_STARTED event...","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":30,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.8085583Z","msg":"Including script file \"scripts\\csvTools.js\"...","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":40,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:41.83856Z","msg":"WARN: Failed to get script asset scripts\\csvTools.js from the sync service. The HTTP status code was: 404. The HTTP body was: Could not find script file scripts\\csvTools.js","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":40,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:42.8640385Z","msg":"WARN: Failed to get script asset scripts\\csvTools.js from the sync service. The HTTP status code was: 404. The HTTP body was: Could not find script file scripts\\csvTools.js","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":50,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:43.8919951Z","msg":"ERROR: java.lang.RuntimeException: Failed to evaluate the \"dataSet\" property of the data-driven test as a JavaScript expression. The expression we evaluated was >>> (function() {\n  $include(\"scripts/csvTools.js\"); // Incluye el script\n\n  var urlData = generateUrl(\"PROD\"); // Obtenemos la URL para el entorno de producci√≥n\n  var loginData = $csvToJson(\"data/py-data/login-mail/login-mail.csv\");\n\n  return {\n    url: urlData,\n    data: loginData\n  };\n})();\n <<<.\r\n\tat org.getopentest.base.TestActor.evalDataSet(TestActor.java:1274)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:2975)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3113)\r\n\tat org.getopentest.Main.main(Main.java:20)\r\nCaused by: java.lang.RuntimeException: Failed to get script file \"scripts\\csvTools.js\".\r\n\tat org.getopentest.base.TestActor.includeScriptFile(TestActor.java:2593)\r\n\tat org.getopentest.base.TestActor.lambda$evalIncludes$6(TestActor.java:1343)\r\n\tat java.base/java.util.Arrays$ArrayList.forEach(Arrays.java:4390)\r\n\tat org.getopentest.base.TestActor.evalIncludes(TestActor.java:1342)\r\n\tat org.getopentest.base.TestActor.access$600(TestActor.java:84)\r\n\tat org.getopentest.base.TestActor$15.accept(TestActor.java:629)\r\n\tat jdk.scripting.nashorn.scripts/jdk.nashorn.internal.scripts.Script$Recompilation$16$1$\\^eval\\_/0x0000000100363c40.L:1(<eval>:2)\r\n\tat jdk.scripting.nashorn.scripts/jdk.nashorn.internal.scripts.Script$Recompilation$15$\\^eval\\_/0x0000000100367c40.:program(<eval>:1)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:655)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:513)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:527)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:456)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:413)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:409)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:162)\r\n\tat java.scripting/javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)\r\n\tat org.getopentest.base.TestActor.evalDataSet(TestActor.java:1272)\r\n\t... 3 more\r\nCaused by: java.lang.RuntimeException: Failed to get script asset scripts\\csvTools.js from the sync service. The HTTP status code was: 404. The HTTP body was: Could not find script file scripts\\csvTools.js\r\n\tat org.getopentest.base.TestActor.getTestAsset(TestActor.java:2431)\r\n\tat org.getopentest.base.TestActor.includeScriptFile(TestActor.java:2587)\r\n\t... 19 more\r\n","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":30,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:43.9753239Z","msg":"Executing segment 1 of test logueo-mail/logueo-mail...","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":30,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:43.9763228Z","msg":"Evaluating action 1/4 (org.getopentest.selenium.NavigateTo) in segment 1 (Navegar a oneApp)...","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":50,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:44.0533238Z","msg":"ERROR: java.lang.Exception: Failed executing test logueo-mail/logueo-mail, segment 1\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2208)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3077)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3113)\r\n\tat org.getopentest.Main.main(Main.java:20)\r\nCaused by: java.lang.RuntimeException: There was an error while executing action no. 1 (org.getopentest.selenium.NavigateTo) in segment 1 of test logueo-mail/logueo-mail.\nThe action definition was:\n{\n  \"action\": \"org.getopentest.selenium.NavigateTo\",\n  \"args\": {\n    \"url\": \"$dataRecord.url\"\n  },\n  \"description\": \"Navegar a oneApp\"\n}\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2142)\r\n\t... 3 more\r\nCaused by: java.lang.NullPointerException\r\n\tat org.getopentest.base.TestActor.executeActionByDef(TestActor.java:1839)\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2092)\r\n\t... 3 more\r\n","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":30,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:44.5767399Z","msg":"Test session has completed","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":10,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:44.5787233Z","msg":"TRACE: Firing TEST_COMPLETED event...","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":10,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:44.5811186Z","msg":"TRACE: Firing SESSION_COMPLETED event...","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":30,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:47:44.5831251Z","msg":"Actor 27587 of type ACTOR1 with no tags is waiting to be acquired by a test session...","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":10,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:49:31.445796Z","msg":"TRACE: Initializing test session information in the \"announce\" method...","v":0}
{"name":"1730836057","hostname":"DESKTOP-8V10OGQ","pid":20940,"level":10,"actorType":"ACTOR1","actorId":"27587","time":"2024-11-05T19:49:31.4477901Z","msg":"TRACE: Session ID in sync server response was 1730836169; actor session ID was null","v":0}
